js/view.js

```diff
@@ -21,6 +21,7 @@ define([
     },
 
     preRender: function() {
+      this.model.set('_items', this.model.getNavigationData());
 
       Adapt.trigger(this.constructor.type + 'View:preRender', this);
```

Comment:
I think you can get rid of this line too now because it's handled by `ComponentView.render` [here](https://github.com/adaptlearning/adapt_framework/blob/018ac92a0b14eb99a5376078089c747c30a34447/src/core/js/views/adaptView.js#L41)

Reply:
I think you can probably delete `getData` from the model unless it's used anywhere else?

Reply:
Thanks guys, though `getData` is still used in the view to get the current state of the buttons

Reply:
@chris-steele does `getData` return different data to `getNavigationData`?

Reply:
I'm with you now - getData dropped

Reply:
```js
          var data = this.model.getData();
          try {
            var execute = new Function(data._items[index]._onClick||'');
```
becomes:
```js
          var data = this.model.getNavigationData();
          try {
            var execute = new Function(data[index]._onClick||'');
```